week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,7 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var total = numberOfStudents + numberOfMentors;
+var perstudent = Math.round((100 * numberOfStudents) / total);
+var permentors = Math.round((100 * numberOfMentors) / total);
```

Comment:
Can you think of a way to use a function to make this more DRY (don't repeat yourself)?